chart/templates/_helpers.tpl

```diff
@@ -33,3 +33,9 @@ chart: {{ .Chart.Name }}-{{ .Chart.Version }}
 heritage: {{ .Release.Service }}
 release: {{ .Release.Name }}
 {{- end }}
+
+{{- define "rancher.systemDefaultRegistry" -}}
+{{- if .Values.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.systemDefaultRegistry -}}
```

Comment:
Since this is user input, ideally we would check for slash somehow, they might add it. Whereas other charts it gets added automatically.

Reply:
the logic is updated to handle this case.